src/components/screens/Authenticator/Authenticator.container.vue

```diff
@@ -45,6 +45,11 @@ export default {
       type: Boolean,
```

Comment:
sms code does not correct process in server:
- you should enable sms otp for your account in vault
- then logout
- and login with google
- when you will receive sms otp modal for, try to reload page
- you will be logged in vault, but you should not

---

and the next:
- you account should use sms otp mode
- when you will receive sms otp modal after login by google, try to enter code
- server return error
```
code: 406
error: "crypto/bcrypt: hashedPassword is not the hash of the given password"
message: "Not Acceptable"
```